### ISSUE TITLE
iOS Podcast application downloads 200% length in audio mode.

### ISSUE THREAD

**Issue by qtlin** [i3arnon/YouCast#11]
When Preferred Quality is set to Audio and generated podcast URL is entered in iOS Podcasts application then downloaded audio file play time is twice as long as original YouTube Video. Second half is silent.

**Comment by i3arnon** [comment]
I'm not sure there's anything to be done about this. YouCast just redirects the podcatcher to the stream YouTube provides.

I'm assuming it's an issue with the app since BeyondPod doesn't do that.

**Comment by HanaBookStory** [comment]
Great Thanks to you i3arnon.
I have same problem about 200% length...
iOS PODCAST app, iTunes, Web.. same problem.

But.. youcast is best application for me. Thank you.

**Comment by JustinReinhart** [comment]
Indeed, it seems that audio downloads at ~200% length and the second half is blank. I hope it is fixable some day.
